src/Models/NonhydrostaticModels/boundary_mass_fluxes.jl

```diff
@@ -17,7 +17,7 @@ const FOBC = BoundaryCondition{<:Open{<:Nothing}} # "Fixed-velocity" OpenBoundar
 @inline north_average(v)  = Average(view(v, :, v.grid.Ny + 1, :), dims=(1, 3)) |> Field
 @inline top_average(w)    = Average(view(w, :, :, w.grid.Nz + 1), dims=(1, 2)) |> Field
 
-@inline function get_boundary_flux(bc, boundary_flux_field)
+function get_boundary_mass_flux(bc, boundary_flux_field)
```

Comment:
dispatch might be simpler than `if` statements here, and will allow easier extension of this function